### ISSUE TITLE
budget() omits names of uncertainties

### ISSUE THREAD

**Issue by heikowestermann** [nrc-cnrc/MetroloPy#29]
Hey,

since 0.6.2 the names of uncertainties are omitted by the budget method.

Here's a minimal example:
```
import metrolopy as uc
x_1 = uc.gummy(42, 0, name='A')
x_2 = uc.gummy(42, 1, name='B')
y = x_1 - x_2
print(y.budget([x_1, x_2]))
```

output for 0.6.1
```
Component Value    u |dy/dx|     s
        B  42.0  1.0     1.0  1.00
        A    42    0     nan   nan
        y   0.0  1.0
```

output for 0.6.2
```
Component Value    u |dy/dx|     s
     x[2]  42.0  1.0     1.0  1.00
     x[1]    42    0     0.0     0
        y   0.0  1.0
```
Please notice the 'Component' column

**Comment by hvparks** [comment]
Oops, I have a fix now in the hotfix branch and will release it in a week or so.  For now as a work around, you can pass names for the x values as parameter in the budget method call:

```
x_1 = uc.gummy(42, 0)
x_2 = uc.gummy(42, 1)
y = x_1 - x_2
print(y.budget([x_1, x_2], xnames=['A','B']))
```

gives:

```
Component Value   u |dy/dx|    s
        B  42.0 1.0     1.0 1.00
        A    42   0     0.0    0
        y   0.0 1.0   
```

**Comment by vysiker** [comment]
Adding the highlighted lines to budget.py shortly after initialization solved this issue for me.
Thanks for this excellent package. I am really apperciating your work and looking for future releases.

![image](https://user-images.githubusercontent.com/99641322/153824598-87423855-8cbc-4b8a-96e8-2cb109c05bb9.png)

**Comment by hvparks** [comment]
This is fixed not in version 0.6.3.
